pyproject.toml

```diff
@@ -1,18 +1,32 @@
+[build-system]
```

Comment:
A lot has been rearranged, and I'm not sure why. Please don't as it makes review hard